package.json

```diff
@@ -1,33 +1,35 @@
 {
-	"name": "showcase",
-	"version": "1.0.0",
-	"private": true,
-	"main": "src/index.jsx",
-	"license": "MIT",
-	"dependencies": {
-		"geopattern": "^1.2.3",
-		"react": "^15.5.4",
-		"react-dom": "^15.5.4",
-		"react-router-dom": "^4.1.2",
-		"redux": "^3.7.2"
-	},
-	"devDependencies": {
-		"babel-core": "^6.24.1",
-		"babel-loader": "^7.0.0",
-		"babel-preset-es2015": "^6.24.1",
-		"babel-preset-react": "^6.24.1",
-		"css-loader": "^0.28.4",
-		"extract-text-webpack-plugin": "2.1.2",
-		"preact": "^8.2.1",
-		"preact-compat": "^3.16.0",
-		"react-hot-loader": "^1.3.1",
-		"style-loader": "^0.18.2",
-		"webpack": "^2.6.1",
-		"webpack-dev-server": "^2.4.5"
-	},
-	"scripts": {
-		"start": "webpack-dev-server --progress --colors --hot --config ./webpack.dev.js",
-		"build": "webpack -p --config ./webpack.prod.js",
-		"webpack-size": "webpack --config ./webpack.prod.js --json | webpack-bundle-size-analyzer"
-	}
+  "name": "showcase",
```

Comment:
Thanks for the PR. 
Just have one issue here. I see a lot of unnecessary diff in this file. It seems that the indentation has messed up. Please fix this.

Reply:
I cannot replicate the issue on my machine by using an IDE or by using the github remote editor. Can you see it locally?

Reply:
I have used 2-tabs in that file whereas your editor has changed it to spaces, thus that extra diff. I recommend you install an `Editorconfig plugin` for your editor as it will help it respect the [.editorconfig](https://github.com/aviaryan/work/blob/master/.editorconfig) file